frontend/src/tests/components/commons/SideNavigation.test.js

```diff
@@ -51,10 +51,7 @@ it("changes page to test1", () => {
     { id: 2, text: TEXT3, body: BODY3 }
   ];
   const wrapper = mount(<SideNavigation navSpecs={SPECS} currentNode={1} />);
-  wrapper
```

Comment:
I think these tests that simulate clicking are actually really useful - can we have it and check that the state updates?

Reply:
Looking into this one: there was a bug with the 'scrollIntoView' function when called from within the test, that's why I took out the button click initially

Reply:
Fixed in e2d53e9 (added a dummy of scrollIntoView; check state rather than presence of id; restored simulate click